### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by davefp** [davefp/handsome#17]
At the time of writing, the following dependencies are outdated:

```
Package                 Current  Wanted  Latest  Location
babel-core                6.7.6  6.18.2  6.18.2  handsome
babel-loader              6.2.4   6.2.7   6.2.7  handsome
babel-preset-es2015       6.6.0  6.18.0  6.18.0  handsome
babel-preset-react        6.5.0  6.16.0  6.16.0  handsome
css-loader               0.23.1  0.23.1  0.25.0  handsome
express                  4.13.4  4.14.0  4.14.0  handsome
ignore-styles             2.0.0   2.0.0   5.0.1  handsome
jquery                    2.2.3   2.2.4   3.1.1  handsome
mocha                     2.5.3   2.5.3   3.1.2  handsome
moment                   2.12.0  2.15.2  2.15.2  handsome
node-sass                 3.4.2  3.11.2  3.11.2  handsome
nyc                       6.6.1   6.6.1   8.4.0  handsome
packery                   2.0.0   2.1.1   2.1.1  handsome
react                    15.0.1  15.3.2  15.3.2  handsome
react-dom                15.0.1  15.3.2  15.3.2  handsome
redis                   2.6.0-1   2.6.3   2.6.3  handsome
request                  2.71.0  2.78.0  2.78.0  handsome
sass-loader               3.2.0   3.2.3   4.0.2  handsome
webpack                 1.12.15  1.13.3  1.13.3  handsome
webpack-dev-middleware    1.6.1   1.8.4   1.8.4  handsome
```

These should be brought up-to-date one by one and tested.

**Comment by erikdstock** [comment]
Hi- I've updated a few of these dependencies, mostly those in the babel/webpack sphere [to Webpack 2]. You didn't list a major version change here but could I make a PR for that once it is cleaned up?

**Comment by davefp** [comment]
@erikdstock absolutely! Thanks for offering

**Comment by davefp** [comment]
Largely resolved by #19
